citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/service/AbstractImportProcessor.java

```diff
@@ -81,45 +65,105 @@ public void init() {
         itemTypeMapping.put(Publication.NEWS_ITEM, ItemType.NEWSPAPER_ARTICLE);
         itemTypeMapping.put(Publication.PROCEEDINGS_PAPER, ItemType.CONFERENCE_PAPER);
         itemTypeMapping.put(Publication.DOCUMENT, ItemType.DOCUMENT);
+        itemTypeMapping.put(Publication.MONOGRAPH, ItemType.BOOK);
+        itemTypeMapping.put(Publication.JOURNAL_ISSUE, ItemType.JOURNAL_ARTICLE);
```

Comment:
this is wrong I believe, an issue is not an article. I think we will simply not deal with those and not import issues.